historical-adherence-query/config.yml

```diff
@@ -0,0 +1,97 @@
+---
+title: Retrieving Notifications from a Historical Adherence Query
+description: Tutorial on how to subscribe to a historical adherence notifications and retrieve the results of a query
+categories:
+- Notifications
+steps:
+- title: Introduction
+  content: |
+    This tutorial shows how to subscribe to historical adherence notifications for a query and retrieve the resulting downloadUrls.
+    Subscribing to notifications is how you are able to retrieve events that include the results of your requests.
+    With historical adherence queries specifically, after your intital request your results will begin processing.
+    After your query has finished processing, an event will be returned that can be caught if subscribed to historical adherence query notifications.
+    The body of this event contains the results of your query, including the downloadUrls with the historical adherence of each user for the range of time you queried.
+- title: Client Credential Authentication
+  content: |
+    Authenticate using Client Credentials
+    For more information on how to authenticate your api client, [here is a useful resource.] (/api/rest/client-libraries/java/index.html#authenticating)
+- title: Retrieving Your User ID
+  content: |
+    Using the UserApi you can retrieve your own userId for the subscription.
+- title: Building the NotificationHandler to listen for historical adherence query events
+  content: |
+    Instantiate a new notification handler and add a listener for the query completed response
+- title: Writing a custom NotificationListener
+  content: |
+    Create a class implementing the NotificationListener<T> interface.
+    A list of mappings of topics to classes for T can be found [here.] (https://github.com/MyPureCloud/platform-client-sdk-java/blob/master/notificationMappings.json)
+    Since we are subscribing to the "v2.users.{myId}.workforcemanagement.historicaladherencequery" topic, the class is WfmHistoricalAdherenceCalculationsCompleteTopicWfmHistoricalAdherenceCalculationsCompleteNotice.
+    The schema for "v2.users.{myId}.workforcemanagement.historicaladherencequery" can be found [here.] (/api/rest/v2/notifications/available_topics.html)
```

Comment:
I just wanted to confirm that the link is indeed correct as it doesn't exactly seem related to me (list of topics vs. schema of the historical adherence query)

Reply:
I think my wording wasn't very clear here, so I will change this, but this is referring to the schema of the body of the event subscribers receive. That page doesn't seem to allow you to link to specific topics to make that clearer, but if you go to the historicaladherencequery topic and expand it, you can see the schema I am talking about here.

Reply:
OK. Yeah i interpreted this to mean the schema of the HA result object.